src/app/zap-templates/zcl/data-model/chip/descriptor-cluster.xml

```diff
@@ -23,6 +23,13 @@ limitations under the License.
     <item name="Revision" type="INT16U"/>
   </struct>
 
+  <struct name="RefSemStruct">
+    <cluster code="0x001d"/>
+    <item name="Name" type="CHAR_STRING"/>
+    <item name="Tag" type="CHAR_STRING"/>
+    <item name="VendorId" type="INT16U"/>
```

Comment:
I don't understand.  This looks nothing like "7.18.2.36. SemanticTagStruct": different number of fields, different field names (except for Tag), different field types, different qualities.  What am I missing here?